### DISCUSSION TITLE
Elsa 3.1.3 publish event

### DISCUSSION THREAD

**Discussion by mremadevi** [elsa-workflows/elsa-core#5317] [Q&A]
Please share a sample JSON workflow which publishes event

**Comment by glime-ai** [comment]
I'm sorry, but I do not have access to a sample JSON workflow that specifically demonstrates the publishing of an event within the Elsa Workflow framework. However, you can create a custom workflow that includes an event publishing activity by using the Elsa Workflow designer or programmatically, and then export it to JSON.

If you are looking to create such a workflow, you can refer to the Elsa documentation and examples provided in the context to understand how to structure your workflow definitions. Once you have created a workflow that publishes an event, you can export it to JSON using the designer's export functionality.

For more detailed guidance on creating workflows with Elsa, you can visit the official Elsa Workflows documentation: https://elsa-workflows.github.io/elsa-core/

If you need further assistance with creating a specific part of the workflow definition, please let me know, and I will do my best to assist you.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).

**Comment by mremadevi** [comment]
closing
